.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -17,7 +17,9 @@ title: 'Bug: '
 
 *What did you expect to happen?*
 
-## Example
+## Example/ Screenshots
+
+*Please provide relevent screenshots  as it not only allows you to help explain the problem more clearly but also allows the reviewer to visually see the bug or its effect*
```

Comment:
Hi Hemant, thank you so much for your contribution! How do you feel about changing the text you wrote a little bit?

```suggestion
*Please provide screenshots to help reviewers  to visualize the bug and how it manifests.*
```